File: services/tts/src/tts/health.py
"""Shared health payload for the tts service."""

from __future__ import annotations

from . import SERVICE_NAME, __version__


def health() -> dict:
    """Return the standard HealthResponse contract payload.

    Shape: {"status": "ok", "service": <name>, "version": <version>}
    """
    return {"status": "ok", "service": SERVICE_NAME, "version": __version__}

File: apps/control_api/src/routes/health.ts
/**
 * Health route. Implements the shared HealthResponse contract:
 *   { status, service, version }
 */

import type { FastifyInstance } from "fastify";

export const SERVICE_NAME = "control_api";
export const SERVICE_VERSION = "0.1.0";

export interface HealthResponse {
  status: "ok";
  service: string;
  version: string;
}

export async function healthRoutes(app: FastifyInstance): Promise<void> {
  app.get("/health", async (): Promise<HealthResponse> => {
    return { status: "ok", service: SERVICE_NAME, version: SERVICE_VERSION };
  });
}
